doc/api/n-api.md

```diff
@@ -490,6 +490,20 @@ Returns `napi_ok` if the API succeeded.
 
 This API returns true if an exception is pending.
 
+### Fatal Errors
+
+In the event of an unrecoverable error in a native module, a fatal error can be
+thrown to immediately terminate the process.
+
+#### napi_fatal_error
+```C
+NAPI_EXTERN NAPI_NO_RETURN void napi_fatal_error(const char* location, const char* message);
+```
+
+- `[in] location`: The location at which the error occurred.
+- `[in] message`: The message associated with the error.
```

Comment:
Make it clear that `location` is `NULL`-able, while `message` isn't. Speaking of that, I'd actually prefer the function signature to be `(message, location)` as it seems customary for `NULL`-able parameters to go last.